installer/functions/task-router/deploy-task-router.ts

```diff
@@ -0,0 +1,124 @@
+// Imports global types
+import '@twilio-labs/serverless-runtime-types';
+import { createTaskQueue, createWorkspace, getAllWorkflows, getAllTaskQueues, createWorkflow, QueueToQueueSid, getAllWorkers } from './task-router-helpers.private';
+import { 
+  WORKSPACE_FRIENDLY_NAME,
+  SCHEDULERS,
+  EDUCATORS,
+  SCHEDULING,
+  EDUCATION,
+  INTAKE_BY_SCHEDULERS,
+  TRANSFER_TO_NURSE_EDUCATOR
+} from './task-router-helpers.private';
+// Fetches specific types
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+import { WorkflowInstance } from 'twilio/lib/rest/taskrouter/v1/workspace/workflow';
+
+export const handler: ServerlessFunctionSignature = async function(
+  context: Context,
+  event: {},
+  callback: ServerlessCallback
+) {
+  
+  const client = context.getTwilioClient();
+  const response = new Twilio.Response();
+  response.appendHeader('Access-Control-Allow-Origin', '*');
+
+  try {
+    // Get the Flex Workspace
+    let flexWorkspace = await client.taskrouter.workspaces
+      .list()
+      .then(workspaces => workspaces.find(workspace => WORKSPACE_FRIENDLY_NAME === workspace.friendlyName));
```

Comment:
i've noticed a mix of `async/await` and then `.then` promisess; might be easier to break these up into separate statements like:

```tsx

    const flexWorkspaces = await client.taskrouter.workspaces.list();
    let flexWorkspace = workspaces.find(workspace => WORKSPACE_FRIENDLY_NAME === workspace.friendlyName));
```